layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -48,43 +61,60 @@ instances:
       tags:
         - 'table:employees'
 ```
 
-### Monitoring multiple databases with DBM and the default integration
-If you wish to collect the generic Postgres integration metrics from logical databases, other than the default `postgres` db, you must list them each in their instance config. **Note**: you should list each logical db *without* setting `dbm: true`, and then add one final instance with `dbm: true` so that we don't "double monitor" the databases. This is because Datadog's Database Monitoring collects metrics from global views like `pg_stat_statements` and `pg_stat_activity`.
+### Monitoring relation metrics for multiple logical databases
+In order to collect relation metrics (such as `postgresql.seq_scans`, `postgresql.dead_rows`, `postgresql.index_rows_read`, and `postgresql.table_size`), the Agent must be configured to connect to each logical database (by default, the Agent only connects to the `postgres` database).
 
+Specify a single "DBM" instance to collect DBM telemetry from all databases. Additionally, specify all logical databases the Agent must connect to. It is important to only have `dbm: true` on one configuration instance per host in order to prevent duplication of metrics.
 ```yaml
 init_config:
 instances:
+  # This instance is the "DBM" instance. It will connect to the
+  # `postgres` database and send DBM telemetry from all databases
   - dbm: true
     host: products-primary.123456789012.us-east-1.rds.amazonaws.com
     port: 5432
     username: datadog
     password: '<PASSWORD>'
+  # This instance only collects data from the `on_sale` database
+  # and collects relation metrics from tables prefixed by "2022_"
   - host: products-primary.123456789012.us-east-1.rds.amazonaws.com
     port: 5432
     username: datadog
     password: '<PASSWORD>'
     dbname: on_sale
+    dbstrict: true
+    relations:
+      - relation_regex: 2022_.*
+        relkind:
+          - r
+          - i
+  # This instance only collects data from the `inventory` database
+  # and collects relation metrics only from the specified tables
   - host: products-primary.123456789012.us-east-1.rds.amazonaws.com
     port: 5432
     username: datadog
     password: '<PASSWORD>'
     dbname: inventory
+    dbstrict: true
+    relations:
+      - relation_name: products
+      - relation_name: external_seller_products
 ```
 
-### Working with hosts through a remote proxy
-When connecting to a database host through a remote proxy, it can be useful to set a custom hostname. Utilize the `reported_hostname` option to override the hostname detected by the agent.
+### Working with hosts through a proxy
+If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry will be tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the agent.
```

Comment:
```suggestion
If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry is tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the Agent.
```